packages/gatsby/src/utils/test-require-error.ts

```diff
@@ -1,6 +1,6 @@
 // This module is also copied into the .cache directory some modules copied there
 // from cache-dir can also use this module.
-export default (moduleName, err) => {
+export const testRequireError = (moduleName, err): boolean => {
```

Comment:
could you type the function inputs? Happy to merge after that! I get that the error thing is weird though because it comes from catch's which can literally have any value so that might need to just be an `any`.

Reply:
:+1: